dango/dex/src/core/liquidity_pool.rs

```diff
@@ -93,6 +102,49 @@ pub trait PassiveLiquidityPool {
         reserve: CoinPair,
         output: Coin,
     ) -> anyhow::Result<(CoinPair, Coin)>;
+
+    /// Reflect the curve onto the orderbook.
+    ///
+    /// ## Inputs
+    ///
+    /// - `base_denom`: The base asset of the pool.
+    /// - `quote_denom`: The quote asset of the pool.
+    /// - `reserves`: The current pool reserves.
```

Comment:
@pacmanifold Our terminology is that a `CoinPair` for a single pool is a "reserve" (singular). An iterator over `CoinPair`s of many pools are "reserves" (plural)